virt/virt-4-11-release-notes.adoc

```diff
@@ -50,6 +51,7 @@ The SVVP Certification applies to:
 //CNV-18162 Move MTV action from VM list page
 
 //CNV-19224 Hypervisor logs
+* You can configure the verbosity level of the `virtLauncher`, `virtHandler`, `virtController`, `virtAPI`, and `virtOperator` pod logs to debug specific components by editing the `HCO` object.
```

Comment:
Suggest a rewording:

You can configure the verbosity level of the `virtLauncher`, `virtHandler`, `virtController`, `virtAPI`, and `virtOperator` pod logs to debug specific components by editing the `HyperConverged` custom resource (CR).